assets/pages/bundles-landing/reducers/reducers.js

```diff
@@ -122,10 +127,15 @@ function contribution(
 
 }
 
+function intCmp(state: string = getQueryParameter('INTCMP', 'gdnwb_copts_bundles_landing_default')): string {
```

Comment:
Could make this a const at the top of the file, but either works, it's personal taste.